app/assets/stylesheets/open_project_global/_variables.sass

```diff
@@ -194,3 +194,9 @@ $user-avatar-mini-border-radius:                50px !default
 $user-avatar-mini-width:                        20px !default
 
 $select-element-padding:                        3px, 24px, 3px, 3px
+
+$homescreen-content-bg-color:                   white
+$homescreen-block-bg-color:                     white
+$homescreen-footer-bg-color:                    #F8F8F8
```

Comment:
I'd suggest to make this `$gray-light` for the time being. I'd like to keep this separate color an option for themes.